lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
This .yaml file will be updated with binding fields in @VelocityKnight's PR.

Reply:
So you should remove the create stuff from yours :)

Reply:
Making a note. I only reviewed category in this. Even at that, the example should be updated in Sarah's